src/css/footer.css

```diff
@@ -43,47 +45,53 @@ footer .footer-address {
   padding-top: 0.5rem;
   padding-bottom: 2rem;
   font-style: normal;
+  color: var(--body-faint-font-color);
 }
 
 footer .footer-links {
   display: flex;
   flex-direction: column;
-  padding-top: 1rem;
-  font-size: 1rem;
   line-height: 1.5rem;
+  margin: 0;
 }
 
 .footer-links-heading {
   display: inline;
-  font-size: 0.75rem;
+  font-size: 12px;
   line-height: 1rem;
   letter-spacing: 0.2em;
   font-family: "IBM Plex Mono", sans-serif;
   margin-bottom: 0;
   font-weight: 300;
+  color: var(--body-font-color);
 }
 
 .footer-links-list {
-  margin-top: calc(1rem * calc(1 - 0));
-  font-size: 0.75rem;
+  font-size: 16px;
   line-height: 1.5rem;
-  color: rgb(102 112 133/1);
+  color: var(--body-faint-font-color);
   display: flex;
   flex-direction: column;
   padding-left: 0;
   gap: 5px;
+  margin: 0;
+}
+
+.footer-links-list-item-detail {
```

Comment:
@JakeSCahill

Reply:
The idea with this class is to make the max-height property. I couldn't do it with the li because that would make the link unclickable.